indexing-solr/src/test/resources/datastream/inventoryMods.xml

```diff
@@ -4,7 +4,19 @@
   <mods:titleInfo>
     <mods:title>Paper title</mods:title>
   </mods:titleInfo>
-  <mods:genre>Dissertation</mods:genre>
+
+  <mods:genre authority="local">Dissertation</mods:genre>
```

Comment:
I'd suggest not using the same value with different attributes (unless you're testing for handling of duplicates), otherwise we can't tell which Dissertation, Paper or Letter genre field was the one that made it through to solr.